v0/iterable/data/IterableTrackPurchaseConfig.json

```diff
@@ -0,0 +1,35 @@
+[
+   
+    {
+      "destKey": "dataFields",
+      "sourceKeys": ["properties"],
+      "required": false
+    },
+    {
+      "destKey": "id",
+      "sourceKeys": [""],
+      "required": false
+    },
+    {
+      "destKey": "createdAt",
+      "sourceKeys": ["originalTimestamp"],
+      "required": false
+    },
+    {
+      "destKey": "campaignId",
+      "sourceKeys": [""],
```

Comment:
where will this value come from?
similarly for templateId

Reply:
I was not sure where in payload these value will come from. so I left it blank. what should I add?

Reply:
we can take from properties for now i think..

Reply:
done